minikube-deploy.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
checking quickly, but I think you're correct. IDE added it as bash. (incoming update)